albumentations/core/bbox_utils.py

```diff
@@ -62,11 +62,15 @@ def __init__(
         label_fields: Optional[Sequence[str]] = None,
         min_area: float = 0.0,
         min_visibility: float = 0.0,
+        min_width: float = 0.0,
+        min_height: float = 0.0,
```

Comment:
Please, copy description of this args from `filter_bboxes` to this docstring too.

Reply:
c850f22